### PR TITLE
byte opcode modification

### PR DESCRIPTION
**Pull request by ashWhiteHat**
Part of https://github.com/privacy-scaling-explorations/zkevm-specs/issues/230

I introduced `Instruction` to `BYTE` opcodes and removed unnecessary files.
The circuit constraints and logics are aligned with zkevm-circuits.
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/evm_circuit/execution/byte.rs#L35

I would appreciate it if you could confirm.
Thank you.

### CODE REVIEW

**Comment by ashWhiteHat** [comment]
should pass test with expected behavior

**Comment by CPerezz** [comment]
@ashWhiteHat is this ready for review???

**Comment by ashWhiteHat** [comment]
@CPerezz Yes, this is ready!

File: specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md [hunk 1]
```diff
@@ -4,7 +4,12 @@
 
 ### EVM behavior
 
-The `CALL` opcode transfers specified amount of ether to callee and creates a new call context and switch to it. This is done by popping serveral words from stack:
+- for opcode `CALL`, transfer specified amount of ether to callee and creates a new call context and switch to it
```
Comment (han0110) [inline specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md:7]: This seems to be outdated, changes in this file could be reverted.

**Review by han0110 — APPROVED** [review]
LGTM! The spec now reflects the circuit's constraints more precisely!

File: specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md (hunk 1 above)
Comment (ashWhiteHat) [reply specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md:7]: I just arranged the layout in order to understand easily.
Do you mean the spec in this file doesn't reflect current spec of zkevm-circuits?

Comment (han0110) [reply specs/opcode/F1CALL_F2CALLCODE_F4DELEGATECALL_FASTATICCALL.md:7]: Ah I see, sorry for misunderstanding.

**Review by CPerezz — APPROVED** [review]
LGTM! Thanks for this work @ashWhiteHat

**Pull request merged**